### ISSUE TITLE
Gambling

### ISSUE THREAD

**Issue by Numbwa** [kolton/d2bot-with-kolbot#1255]
whenever my guy goes to gamble i get the error "pickit error in runeword config" where do i find this config? i've attached a pastebin of my char config, Let me know if a pastebin of my pickit or shopbot will be needed
[XM5zfJaz.txt](https://github.com/kolton/d2bot-with-kolbot/files/2884910/XM5zfJaz.txt)

**Comment by DarkHorseDre** [comment]
Not sure, whats the full line of the error? usually has filenames and line numbers - link a screenshot of the console with error plx

**Comment by Numbwa** [comment]
sorry for the late answer, dont have enough money atm for it to go gamble as i just manual gambled it all. but the error message is just "pickit error in runeword config" no numbers or anything which is why im confused what the error means. Normally it's easy to go correct cuz of the filenames and line numbers

**Comment by DarkHorseDre** [comment]
yeah i thought maybe you were making runewords but you have it disabled in the config.  I think I've seen that error before but cant remember..

**Comment by Numbwa** [comment]
heres a screenshot of the console.
https://gyazo.com/ea579e002c10b416a37e508035529a93

**Comment by DarkHorseDre** [comment]
ok so its an item error of NTIP.CheckItem of NTItemParser.dbl

please link Tester.nip and LLD.nip

**Comment by Numbwa** [comment]
Heres my pickit tester.nip
[pghPpdSz.txt](https://github.com/kolton/d2bot-with-kolbot/files/2886099/pghPpdSz.txt)

and this is LLD.nip
[Zm1j8Eqe.txt](https://github.com/kolton/d2bot-with-kolbot/files/2886102/Zm1j8Eqe.txt)

**Comment by Numbwa** [comment]
think the problem lies in this line - Scripts.ShopBot = true; // Shopbot script. Automatically uses shopbot.nip and ignores other pickits.
		// Supported NPCs: Akara, Charsi, Gheed, Elzix, Fara, Drognan, Ormus, Asheara, Hratli, Jamella, Halbu, Anya. Multiple NPCs are also supported, example: [NPC.Elzix, NPC.Fara]
		// Use common sense when combining NPCs. Shopping in different acts will probably lead to bugs. - when switched to false it will start gambling for the item i want without giving me any errors,

**Comment by DarkHorseDre** [comment]
true!  have you fixed it then?
